src/applications/financial-status-report/config/form.js

```diff
@@ -1082,6 +1082,7 @@ const formConfig = {
           uiSchema: {},
           schema: { type: 'object', properties: {} },
           depends: () => false, // accessed from otherExpensesSummary
+          returnUrl: 'other-expenses-summary',
```

Comment:
Did we end up using the returnUrl anywhere? I couldn't find it referenced in the files I thought it could be in.

Reply:
@amponce this is newer functionality implemented in #24343 , it's an extra safety net if the save in progress return url is a page that depends is false for